### DISCUSSION TITLE
Does this also work for the IONIQ5?

### DISCUSSION THREAD

**Discussion by ZuinigeRijder** [ZuinigeRijder/hyundai_kia_connect_monitor#1] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

**Comment by beachbum-FL** [comment]
does this monitor alwo work for the IONIQ5 ?

**Reply by ZuinigeRijder** [reply]
Yes, I have an IONIQ 5 and using it.  But Hyundai did a screw up in Europe with NE_E.EUR.S5W_L.001.001.221129 infotainment update, so the odometer is not reported correctly anymore. I did mail this problem to Hyundai in Europe, so I hope they fix this soon. This is the mail I did send:

Annoying Bluelink odometer bug after installing update 221129 for IONIQ 5

I'm having an annoying Bluelink odometer problem after installing update NE_E.EUR.S5W_L.001.001.221129 on December 6th for IONIQ5:
[https://update.hyundai.com/EU/E1/update ... w/623/pop2](https://update.hyundai.com/EU/E1/updateNoticeView/623/pop2)

I use the latest Bluelink App on Android, 2.0.7, but I think this problem is NOT related to the Bluelink App, but because of the infotainment update on my IONIQ 5:
NE_E.EUR.S5W_L.001.001.221129

After turning on the car, the odometer is fetched once properly using the Bluelink App, after that it fetches an old value, in my case 20795.4 km, while the current value is 20826 km. Odometer value 20795.4 km was (coincidentally?) the odometer value just before the update installation.

How can I reproduce it?
- Turn on the car and wait for everything to start, including loading the map
- Make sure the Bluelink app is completely closed
- Start the Bluelink App and go to: Status -> Vehicle Report
- The odometer in the Bluelink app shows the correct value.
- Turn off the car and wait 15 minutes.
- Make sure the Bluelink app is completely closed
- Start the Bluelink App and go to: Status -> Vehicle Report
- The odometer in the Bluelink app shows the wrong odometer

Note: I am not forcing data updates with the Bluelink app

My suspicion is that the IONIQ 5 is sending the wrong old odometer to the Bluelink server when it's turned off.
Others confirmed the same problem:

Yes, the new update is bullshit, see my graphic from my smart home. Since installation, the correct status is only transmitted when driving, then the status during the update... Really great...


However, since the Bluelink app can display the correct odometer when the car is on and the data is always transmitted via the Bluelink server, this can certainly be solved on the Bluelink server side and does not require a new update in the vehicle.

Can this be solved on the Bluelink Server soon and in future Car Infotainment updates?
